plugin/open_url.vim

```diff
@@ -3,13 +3,13 @@ if exists('g:loaded_open_url')
 endif
 let g:loaded_open_url = 1
 
-function! s:SetGlobalOpt(opt, val)
-  if !exists('g:'.a:opt)
-    let g:{a:opt} = a:val
-  endif
-endfunction
```

Comment:
This is a nice helper, it may not seen useful but results in much cleaner plugin configuration setup which is both easier to read and update. Kindly do not remove this.

Reply:
OK, this is a nice helper. 

> it may not seen useful

So let's make it useful ;)